htdocs/core/modules/modTakePos.class.php

```diff
@@ -53,7 +53,7 @@ public function __construct($db)
 		// It is used to group modules by family in module setup page
 		$this->family = "portal";
 		// Module position in the family on 2 digits ('01', '10', '20', ...)
-		$this->module_position = '90';
+		$this->module_position = 90;
```

Comment:
Are you sure. The title say "module_position" is on 2 digits, '01', '02', ...
Same letter, example are using strings

Reply:
from DolibarrModules.class.php:

/**
 * @var int Module position
 * @since 3.9.0
 */
public $module_position=500;

Reply:
We should replace with 
module_position='99' into DolibarrModules i think.

Reply:
@eldy I think position is integer because it' integer in db